docs/docs/thinking-in-react.md

```diff
@@ -93,8 +93,8 @@ Think of all of the pieces of data in our example application. We have:
 Let's go through each one and figure out which one is state. Simply ask three questions about each piece of data:
 
   1. Is it passed in from a parent via props? If so, it probably isn't state.
-  2. Does it change over time? If not, it probably isn't state.
-  3. Can you compute it based on any other state or props in your component? If so, it's not state.
+  2. Does it remain the same over time? If so, it probably isn't state.
```

Comment:
"remain the same over time" feels a bit odd to me, though I can't put my finger on exactly why.  I read it and I'm like "what does that mean?"

Perhaps: "Is it a constant value that never changes?  If so, it probably isn't state."